tests/e2e/pkg/kubernetes.go

```
@@ -324,7 +354,7 @@ func IsVerrazzanoMinVersion(minVersion string) (bool, error) {
 func IsProdProfile() bool {
```

Nothing to do with your change but these should probably return the error instead of defaulting to false.